### DISCUSSION TITLE
TypeError: a string or stream input is required

### DISCUSSION THREAD

**Discussion by camfoy** [AltimateAI/vscode-dbt-power-user#1242] [Q&A]
I am getting a strange error when opening my dbt repo in a Dev Container. Note that my dbt configs work perfectly fine before installing Power User for dbt Core. Only after installing the extension do I start getting dbt errors.

![image](https://github.com/AltimateAI/vscode-dbt-power-user/assets/45903536/09676796-f643-4965-9aa0-0efc0b6cb6ad)

If I click on the error, I'm just taken to the dbt_project.yml. It was correct and working before the Power User install, but now it looks like this everywhere, and every red line just tells me this upon hovering over it: "dbt configuration is invalid : a string or stream input is required".

![image](https://github.com/AltimateAI/vscode-dbt-power-user/assets/45903536/b67f49df-e5ec-4423-af5c-7015366f05c7)

I've had no luck searching up this error online for the past two days. My container works without the extension, but I want the extension installed to help newer users (or generally users other than me) at my company. Does anyone know what's happening here?

**Comment by anandgupta42** [comment]
Hey @camfoy 

We need help replicating the issue. Can you please go ahead and [file a bug report](https://github.com/AltimateAI/vscode-dbt-power-user/issues/new/choose) with instructions on how it can be replicated?

**Reply by felixs7** [reply]
I am getting the same issue. This also appears when just copying a previously working dbt_project.yml file.

**Reply by anandgupta42** [reply]
Can you please go ahead and [file a bug report](https://github.com/AltimateAI/vscode-dbt-power-user/issues/new/choose) with instructions on how it can be replicated?

**Reply by kyleburke-meq** [reply]
Was there an issue open for this? I am getting the same error

**Comment by mdesmet** [comment]
Is it possible to provide a full stacktrace by using [this Python file](https://github.com/AltimateAI/vscode-dbt-power-user/blob/master/dbt_core_integration.py) and running following code:

```
from dbt_core_integration import *

project = DbtProject(project_dir="<your project dir>", profiles_dir="<your profiles dir>", target_path="<your target path>")

project.init_project()
```

**Comment by alittlesliceoftom** [comment]
Getting the same issue.

**Reply by alittlesliceoftom** [reply]
Well similar: 

"Error while compiling: dbt configuration is invalid : a string or stream input is required"

**Reply by anandgupta42** [reply]
@alittlesliceoftom Can you run the python file mentioned [here](https://github.com/AltimateAI/vscode-dbt-power-user/discussions/1242#discussioncomment-10372806) and share the result with us?

**Comment by anandgupta42** [comment]
@camfoy, @alittlesliceoftom, @kyleburke-meq, @felixs7, could you please provide the version of dbt core you are using when encountering this issue?

**Reply by kyleburke-meq** [reply]
For me it was on 0.44.1. I downgraded to 0.43.3 and it works now.

**Reply by anandgupta42** [reply]
@kyleburke-meq What is the version of dbt core you are using?

**Reply by kyleburke-meq** [reply]
@anandgupta42 1.4.5

**Reply by yvetteyuan23** [reply]
experiencing the same issue with 1.2.2

**Comment by anandgupta42** [comment]
We have just released a fix. Please update to version 0.44.3 and confirm if the issue is resolved.

**Reply by kyleburke-meq** [reply]
This appears to work for me. Do you mind explaining what the issue was or point to the PR that fixed it?

**Reply by felixs7** [reply]
This works, thanks for fixing!
